chrome/thunderlink/content/preferences.js

```diff
@@ -49,10 +49,25 @@ var ThunderLinkPrefNS = {
       vbox.appendChild(csLabel);
       vbox.appendChild(csTextbox);
 
+      var clipboardCheckbox = window.document.createElementNS(XUL_NS, "checkbox");
+      clipboardCheckbox.setAttribute("id", "thunderlink-prefCustomTlString" + cstrnum + "-clipboard-checkbox");
+      clipboardCheckbox.setAttribute("checked", ThunderLinkPrefNS.GetPreferenceValue("custom-tl-string-" + cstrnum + "-clipboard-checkbox", "bool"));
+      clipboardCheckbox.setAttribute("preference", "prefs_customTlString" + cstrnum + "tagcheckbox");
```

Comment:
I think this one and the tagcheckbox below are reversed ? when I open the preferences, the tag checkboxes are all checked instead of the clipboard checkboxes

Reply:
You are correct. There is error on lines 55 and 70.

Reply:
Yes, you are correct. The lines 55 and 70 need to be switched.